preamble/syllabus.qmd

```diff
@@ -7,45 +7,55 @@
 Our overall learning outcome for the course is:
 
 1.  Describe some core features of effective team-based, collaborative
-    workflows and identify the components that make these workflows
-    effective compared to other workflows. Then use Git and GitHub with
-    approaches that strongly support effective collaboration.
+    workflows and use Git and GitHub with approaches that support
+    effective collaboration.
 
 This outcome is broken down into specific learning objectives that will
 be addressed within the individual sessions:
 
-1.  Explain what humans, both psychologically and 
-    organizationally, need in order to work well together as a team.
+1.  Explain what humans, both psychologically and organizationally, need
+    in order to work well together as a team.
 2.  Discuss the different ways people work together as a team and
-    identify how some of these ways work better than others for effective
-    teamwork.
+    identify how some of these ways work better than others for
+    effective teamwork.
 3.  Describe how the widely used Git and GitHub are used for
     collaboration, explain their biggest strengths and weaknesses
     compared to alternatives, and review the basics of using Git and
     GitHub.
 4.  Set up a project on GitHub (called a repository) and apply some key
     settings on GitHub to improve collaboration and teamwork.
 5.  Differentiate between contributor and reviewer/admin roles in a team
-    and why they should be dynamic and explicit.
+    and explain why they should be dynamic and explicit.
 6.  Create and use a task list (called issues) to assign team members to
     tasks that they are responsible for.
 7.  Apply a contributor workflow that involves:
-		1. selecting an issue to work on,
-		2. creating an isolated section of a repository (called
-    	 branches), 
-    3. making small and distinct changes to files (known as
-    	 atomic commits) with clear messages explaining why or what
-    	 was changed, and 
-    4. contributing those changes into the main branch
-       of a repository (called pull requests).
+    1.  selecting an issue to work on,
+    2.  creating an isolated section of a repository (called branches),
+    3.  making small and distinct changes to files (known as atomic
+        commits), and
+    4.  contributing those changes to the project (called pull
+        requests).
 8.  Apply a reviewer/admin workflow that involves:
-		1. reviewing a pull request with changes, 
-		2. giving suggestions and feedback, and 
-		3. identifying how (and if) the changes should be merged 
-			 back into the main branch of a repository.
+    1.  reviewing a pull request,
+    2.  giving suggestions and feedback, and
+    3.  identifying how (and if) the changes should be merged into
+        project.
 
 ## Is this course for you? {#sec-is-it-for-you}
 
+This course will cover:
+
+-   Using Git and GitHub with VS Code and the GitHub web interface
```

Comment:
I added that we'll use the GitHub web interface - bc that's also what we do with all our work, and I think it'll make sense to use it in this course as well 👍